project/src/main/kotlin/org/ionproject/core/calendar/ICalendarHttpMessageConverter.kt

```diff
@@ -14,10 +14,10 @@ import org.springframework.http.converter.AbstractGenericHttpMessageConverter
 import java.io.PrintWriter
 import java.io.Writer
 import java.lang.reflect.Type
+import java.nio.charset.StandardCharsets
```

Comment:
Where is this import being used?

Reply:
It's not used. It's removed by PR #281 . For simplicity, it was not changed in this pull request to avoid conflicts.